lib/components/godoc-panel-view.js

```diff
@@ -0,0 +1,39 @@
+/** @babel */
+/** @jsx etch.dom */
+
+import etch from 'etch'
+
+export default class GodocPanelView {
```

Comment:
I could be talked into this.  I don't particularly like inheritance if it isn't necessary, so when this was in my own package I opted not to create the superclass.

https://github.com/atom/etch#what-about-a-component-superclass